### DISCUSSION TITLE
SMPlayer new user - easy questions

### DISCUSSION THREAD

**Discussion by VideoBob** [smplayer-dev/smplayer#799] [Q&A]
Forgive me for asking obvious questions, but I can't find the answers.

1) I installed SMPlayer 21.10.0 (rev 10000 64-bit) on Win11 from the Microsoft store.  It works.  But I can't find smplayer.exe on my local machine.  Where is the program stored?

2) Can I configure SMPlayer to automatically load the .srt file that is in the same drive as the video I selected?  VLC does that.  

Thank you.

**Comment by smplayer-dev** [comment]
> Forgive me for asking obvious questions, but I can't find the answers.
> 
> 1. I installed SMPlayer 21.10.0 (rev 10000 64-bit) on Win11 from the Microsoft store.  It works.  But I can't find smplayer.exe on my local machine.  Where is the program stored?

Applications from the Microsoft store are installed in hidden folders.
https://answers.microsoft.com/en-us/windows/forum/all/microsoft-store-executable-file-location/fa06d159-c8ba-43f8-8a5b-2a30b75e17c3

> 2. Can I configure SMPlayer to automatically load the .srt file that is in the same drive as the video I selected?  VLC does that.
> 
> Thank you.

There's an autoload option in preferences -> subtitles.
